.travis.yml

```diff
@@ -70,9 +70,32 @@ jobs:
         - pip install tox
         # Wait for server to be available
         - until $(curl --output /dev/null --silent --head --fail -k https://example.com:4000); do printf '.'; sleep 5; done
+    - stage:
+      <<: *tox
+      addons:
+        firefox: latest-nightly
+        hosts: example.com
+      env: ALLOWED_TO_FAIL=1 TOXENV=discopane-ui-tests GECKODRIVER=0.20.1 MOZ_HEADLESS=1 PATH=./node_modules/.bin:$PATH
```

Comment:
If this is allowed to fail, no one will care about it. I don't think it will be helpful to add an additional stage here.

Reply:
Well, I will care about it. Otherwise, how do we know if this gets fixed on Nightly?

Reply:
> Otherwise, how do we know if this gets fixed on Nightly?

The person looking into fixing the tests for Nightly would be running it locally.

But, sure, that's fine if you want to leave in a new stage temporarily while someone works on a fix.